css/main.css

```diff
@@ -1165,6 +1165,19 @@ li.tg-rss a{background:#ff6600;}
 	width:38px;
 	line-height:38px;
 }
+
+@media screen and (max-width: 767px) {
```

Comment:
This `max-width` 767 px and `min-width` 768 px is really odd.

Can we improve it?